eng/pipelines/templates/stages/archetype-java-release.yml

```diff
@@ -152,14 +152,19 @@ stages:
               environment: github
               dependsOn: PublishPackage
               variables:
-                - template: ../variables/globals.yml
+                CIConfigs: "{'targets':[{'path_to_config':'package.json','mode':'Preview','monikerid':'1'},{'path_to_config':'package.json','mode':'Latest','monikerid':'0'}]}"
```

Comment:
This CIConfig is a little hard to read/parse inline in the yml. Do we have any better options to pass this data? Perhaps we could also put it as a config file that we read.

Reply:
Yeah. I'll shift it to a config file location in the repo in another PR.